content/faq/invites.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How does the invite program work?
+category: getstarted
+---
+
+You can earn LBRY Credits (LBC) for inviting others to use LBRY as well as letting us know who invited you. You can invite new users to LBRY via email or a customized URL on the Invites page of LBRY Desktop or lbry.tv. This page will also show the status of existing invites.
+
+*Note: Referral redemptions are typically limited to 20, but increase as you have more verified subscribers on LBRY. See the [limits section below](#limits)*
```

Comment:
tempted to leave this even more vague for abuse reasons, but also okay with this